### PR TITLE
feature(setup-wallet): restore hw legacy funnel restyling

### PR DESCRIPTION
**Pull request by banklesss**
## Description / Change(s) / Related issue(s)

Restyling if the restore hw wallet funnel.

##### Ticket

YOMO-1818

### CODE REVIEW

File: apps/wallet-mobile/src/legacy/HW/LedgerConnect/DeviceItem/DeviceItem.tsx
Comment (banklesss) [inline apps/wallet-mobile/src/legacy/HW/LedgerConnect/DeviceItem/DeviceItem.tsx:1]: legacy guys. will be removed soon

File: apps/wallet-mobile/src/legacy/HW/LedgerConnect/LedgerConnect.tsx
Comment (banklesss) [inline apps/wallet-mobile/src/legacy/HW/LedgerConnect/LedgerConnect.tsx:1]: legacy guys. will be removed soon

**Review by michalszorad — APPROVED** [review]

File: packages/setup-wallet/src/translators/reactjs/state/state.ts [hunk 1]
```diff
@@ -118,7 +118,7 @@ export enum SetupWalletActionType {
   PublicKeyHexChanged = 'publicKeyHexChanged',
   PathChanged = 'pathChanged',
   HwDeviceInfoChanged = 'hwDeviceInfoChanged',
-  SetUpTypeChanged = 'setUpTypeChanged',
+  SetUpTypeChanged = 'setupTypeChanged',
```
Comment (michalszorad) [inline packages/setup-wallet/src/translators/reactjs/state/state.ts]: Is it worth renaming the keys as well?

File: apps/wallet-mobile/src/features/SetupWallet/illustrations/LedgerCheckIllustration.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
+import * as React from 'react'
+import Svg, {Circle, Defs, LinearGradient, Path, Stop, SvgProps} from 'react-native-svg'
+
+export const LedgerCHeckIllustration = (props: SvgProps) => {
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/SetupWallet/illustrations/LedgerCheckIllustration.tsx]:
```suggestion
export const LedgerCheckIllustration = (props: SvgProps) => {
```

File: apps/wallet-mobile/src/features/SetupWallet/useCases/RestoreHwWallet/CheckNanoXScreen.tsx [hunk 3]
```diff
@@ -3,13 +3,16 @@ import {useSetupWallet} from '@yoroi/setup-wallet'
...
 import {SafeAreaView} from 'react-native-safe-area-context'
 
-import image from '../../../../assets/img/ledger_1.png'
-import {BulletPointItem, Button, ProgressStep, Spacer, Text} from '../../../../components'
+import {BulletPointItem, Button, Text} from '../../../../components'
+import {Space} from '../../../../components/Space/Space'
+import {StepperProgress} from '../../../../components/StepperProgress/StepperProgress'
 import {confirmationMessages, ledgerMessages} from '../../../../kernel/i18n/global-messages'
 import {SetupWalletRouteNavigation} from '../../../../kernel/navigation'
+import {LedgerCHeckIllustration} from '../../illustrations/LedgerCheckIllustration'
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/SetupWallet/useCases/RestoreHwWallet/CheckNanoXScreen.tsx]: next line `CH` -> `Ch`

File: apps/wallet-mobile/src/features/SetupWallet/useCases/RestoreHwWallet/SaveNanoXScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+    root: {
+      backgroundColor: color.bg_color_high,
+      ...atoms.justify_between,
+      ...atoms.px_lg,
+    },
+    safeAreaView: {
+      ...atoms.pb_lg,
+    },
+    info: {
+      flexDirection: 'row',
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/SetupWallet/useCases/RestoreHwWallet/SaveNanoXScreen.tsx]:
```suggestion
      ...atoms.flex_row,
```

File: apps/wallet-mobile/src/legacy/HW/LedgerConnect/DeviceItem/DeviceItem.tsx [hunk 5]
```diff
@@ -43,18 +47,16 @@ const useStyles = () => {
...
-      marginVertical: 8,
-      marginHorizontal: 16,
-      borderColor: color.secondary_c500,
+      borderColor: color.primary_c500,
       borderWidth: 1,
       borderRadius: 8,
       flexDirection: 'row',
       alignItems: 'center',
-      justifyContent: 'space-between',
+      justifyContent: 'center',
```
Comment (stackchain) [inline apps/wallet-mobile/src/legacy/HW/LedgerConnect/DeviceItem/DeviceItem.tsx]:
```suggestion
      ...atoms.justify_center,
```

File: apps/wallet-mobile/src/features/SetupWallet/useCases/RestoreHwWallet/CheckNanoXScreen.tsx (hunk 3 above)
Comment (banklesss) [inline apps/wallet-mobile/src/features/SetupWallet/useCases/RestoreHwWallet/CheckNanoXScreen.tsx]:
```suggestion
import {LedgerCheckIllustration} from '../../illustrations/LedgerCheckIllustration'
```

**Review by stackchain — APPROVED** [review]
lg, few typos + theming

File: apps/wallet-mobile/src/features/SetupWallet/useCases/RestoreHwWallet/CheckNanoXScreen.tsx [hunk 6]
```diff
@@ -29,35 +32,40 @@ export const CheckNanoXScreen = () => {
...
       </Actions>
     </SafeAreaView>
   )
 }
 
+const Illustration = () => {
+  const styles = useStyles()
+  return (
+    <View style={styles.illustration}>
+      <LedgerCHeckIllustration />
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/SetupWallet/useCases/RestoreHwWallet/CheckNanoXScreen.tsx]:
```suggestion
      <LedgerCheckIllustration />
```

**Pull request merged**